department-of-veterans-affairs/va.gov-cms
# VACMS-1117: update mismatched entities and fields

## Description

See #1117 

## Testing done

- Pulled new prod DB, ran database update, verified that status errors are cleared
- Added test to confirm that no Drupal status errors are present

## QA steps

As an admin user:
1. Go to the status reports and confirm that errors are shown
1. Run database updates
1. Then validate Acceptance Criteria from issue
- [x] No errors are shown on the Drupal status report


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
